src/main/cpp/blaze_util_posix.cc

```diff
@@ -217,6 +218,29 @@ string GetHomeDir() { return GetPathEnv("HOME"); }
 
 string GetJavaBinaryUnderJavabase() { return "bin/java"; }
 
+string Which(const string& executable) {
+  string path = GetPathEnv("PATH");
```

Comment:
Take the chance to make all these local variables `const`.

Reply:
Done.